### PR TITLE
Getting started guide draft

### PR DESCRIPTION
**Pull request by jfan-dolby**
Initial PR, more changes to come. Please only review, do not approve just yet. 
For copyeditor, please directly push to the branch (doc-update)

### CODE REVIEW

**Review by 4thlabs — APPROVED** [review]

**Review by FabienLavocat — DISMISSED** [review]

**Review by FabienLavocat — DISMISSED** [review]

File: docs/documentation/unity/getting-started/initialization.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This guide demonstrates how to initialize, authenticate, and connect to a demo conference. The demo conference has several bots injecting audio, making it easier to test the connection and project setup during the prototyping phase. 
 
-To start fresh, you can create a new project from Unity Hub using the `Third Person` template. The following video shows the workflow:
+To start fresh, you can create a new project from the Unity Hub. This guide use the `Third Person` template in Unity version 2021.3.6f1 LTS. The following video shows the workflow:
```
Comment (FabienLavocat) [inline docs/documentation/unity/getting-started/initialization.md:5]: `use` => `uses`

**Review by FabienLavocat — CHANGES_REQUESTED** [review]

File: docs/documentation/unity/getting-started/initialization.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+To add the SDK to Unity, follow these steps:
+
+1. Create an empty game object in your scene and provide a name for the object. In this example, we call the object `AppManager`:
+
+2. Select the created game object and select `Component` > `DolbyIO` > `DolbyIO Manager` from the Unity menu. This step adds the `DolbyIO Manager` to the game object.
+
+## Set up the initial visual scripting
+1. Select the `AppManager` object and add a `Script Machine` by choosing `Component` > `Visual Scripting` > `Script Machine` from the Unity menu.
+
+2. Add a new script to the `Script Machine` component. In the component click the `New` button and name and save your new script. Start editing the script with the `Edit Graph` button.
```
Comment (FabienLavocat) [inline docs/documentation/unity/getting-started/initialization.md:18]: `In the component click, the`

File: docs/documentation/unity/getting-started/initialization.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+## Initialization
+
+This guide demonstrates how to initialize, authenticate, and connect to a demo conference. The demo conference has several bots injecting audio, making it easier to test the connection and project setup during the prototyping phase. 
+
+To start fresh, you can create a new project from the Unity Hub. This guide use the `Third Person` template in Unity version 2021.3.6f1 LTS.
```
Comment (FabienLavocat) [inline docs/documentation/unity/getting-started/initialization.md:5]: `This guide uses the`

File: docs/documentation/unity/getting-started/initialization.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    Add these connections:
+
+    * Connect the `On Start` event to the input trigger of the `Initialize` node
+    * Connect the output of the `String` node to the `Access Token` input of the `Initialize` node
+    * Connect the `Initialize` event to the input trigger of the `Demo Conference` node
+
+> To keep things simple for now, set the `Spatial Audio` option to `none` in the `Demo Conference` node. This is because unless you have provided your spatial position, the default platform behavior for the spatial audio conference is **no rendering**, which means you will not hear any audio. Setting the `Spatial Audio` flag to `none` informs the platform to always render the audio for the local participant. In the next sections we will explain how to setup the spatial audio conference.
+
+## Step 4. Set up the Client Access Token
+You should have signed up in Dolby.io by now. In the app that is automatically created for you in the Dolby.io dashboard, acquire a temporary Client Access Token and paste in the `String` literal node. For security reasons, the token you acquired will expire in 12 hours. You will have to provide a new token after the expiration.
```
Comment (FabienLavocat) [inline docs/documentation/unity/getting-started/initialization.md:38]: `and paste it in the`

Comment (FabienLavocat) [inline docs/documentation/unity/getting-started/initialization.md:38]: `the token you acquired expires after 12 hours`

**Review by FabienLavocat — CHANGES_REQUESTED** [review]

**Review by FabienLavocat — APPROVED** [review]

**Review by 4thlabs — APPROVED** [review]

**Pull request merged**